### ISSUE TITLE
I don't know how to solve this problem.

### ISSUE THREAD

**Issue by EungGo** [cfzd/Ultra-Fast-Lane-Detection#182]
When I run demo.py, this problem is happening. If you can solve this problem, I want to teach how to solve this problem.
![1](https://user-images.githubusercontent.com/80038814/110103965-e9e6c880-7de9-11eb-9a48-c5c1e19b36bb.PNG)

Problem : 
usage: demo.py [-h] [--local_rank LOCAL_RANK] [--dataset DATASET]
               [--data_root DATA_ROOT] [--epoch EPOCH]
               [--batch_size BATCH_SIZE] [--optimizer OPTIMIZER]
               [--learning_rate LEARNING_RATE] [--weight_decay WEIGHT_DECAY]
               [--momentum MOMENTUM] [--scheduler SCHEDULER]
               [--steps STEPS [STEPS ...]] [--gamma GAMMA] [--warmup WARMUP]
               [--warmup_iters WARMUP_ITERS] [--backbone BACKBONE]
               [--griding_num GRIDING_NUM] [--use_aux USE_AUX]
               [--sim_loss_w SIM_LOSS_W] [--shp_loss_w SHP_LOSS_W]
               [--note NOTE] [--log_path LOG_PATH] [--finetune FINETUNE]
               [--resume RESUME] [--test_model TEST_MODEL]
               [--test_work_dir TEST_WORK_DIR] [--num_lanes NUM_LANES]
               config
demo.py: error: the following arguments are required: config

**Comment by cfzd** [comment]
@EungGo 
You need to run the python code with a config like this:
```
python demo.py configs/culane.py
```
